agent/csi/libstorage/csi_service_libstorage.go

```diff
@@ -292,6 +299,10 @@ func (d *driver) ControllerUnpublishVolume(
 		return nil, err
 	}
 
+	d.pubInfoRWL.Lock()
+	defer d.pubInfoRWL.Unlock()
+	delete(map[string]*pubInfoCache{}, volumeID)
```

Comment:
Hi @codenrhoden,

Shouldn't this line be:

```go
delete(d.pubInfo, volumeID)
```

Reply:
probably. I think I got that from Atom's autofill.  Sad, but I've never had to delete an item from a map before. :)  I can make that change.

Reply:
Let me restate it -- My suggestion wasn't really a suggestion. I was being polite :)

Reply:
thanks for catching this. silly mistake

Reply:
No worries. Good catch on the Atom autofill doing the wrong thing. That's weird. And I guess I see why no linter or compiler caught it. You're technically deleting a key/value pair from a map. It's just a brand new map :)